modules_core/com.smf.jobs.defaults/src/com/smf/jobs/defaults/ProcessShipment.java

```diff
@@ -71,7 +71,7 @@ protected ActionResult action(JSONObject parameters, MutableBoolean isStopped) {
     return result;
   }
 
-  private void massiveMessageHandler(ActionResult result, List<ShipmentInOut> inputs, int errors, int success) {
+  public void massiveMessageHandler(ActionResult result, List<ShipmentInOut> inputs, int errors, int success) {
```

Comment:
Just a little tip. 

If you wish to have a method not be accessible to other modules, but you still want to use it in tests, you can use the package-only scope definition for a method. To do this, simply don't add nothing before the method's return type declaration. That is, not public, nor private, nor protected.

`void massiveMessageHandler(ActionResult result, List<ShipmentInOut> inputs, int errors, int success) { ...`